charts/dependency-track/templates/backend/deployment.yaml

```diff
@@ -61,6 +61,28 @@ spec:
         - name: api
           containerPort: 8080
           protocol: TCP
+        {{- if .Values.apiserver.livenessProbe.enabled }}
+        livenessProbe:
+          httpGet:
+            port: {{ .Values.apiserver.livenessProbe.port | default 8080 }}
+            path: {{ .Values.apiserver.livenessProbe.path | default "/" }}
+          initialDelaySeconds: {{ .Values.apiserver.livenessProbe.initialDelaySeconds | default 45 }}
```

Comment:
drop the default here and just move the values to `values.yaml` which is then easier to see and understand.

Can you also bump the chart version x.y.Z

Reply:
Copy that.